build.sh

```diff
@@ -166,11 +166,35 @@ __rootbinpath="$__scriptpath/bin"
 __msbuildpackageid="Microsoft.Build.Mono.Debug"
 __msbuildpackageversion="14.1.0.0-prerelease"
 __msbuildpath=$__packageroot/$__msbuildpackageid.$__msbuildpackageversion/lib/MSBuild.exe
-__BuildArch=x64
 __buildmanaged=false
 __buildnative=false
 __TestNugetRuntimeId=win7-x64
 
+# Use uname to determine what the CPU is.
+CPUName=$(uname -p)
```

Comment:
Maybe, I took the above from my initial version which is still as is on CoreRT, but I guess CoreCLR was updated later. I'll have a look.